lotus-model.gemspec

```diff
@@ -25,4 +25,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler',  '~> 1.6'
   spec.add_development_dependency 'minitest', '~> 5'
   spec.add_development_dependency 'rake',     '~> 10'
+  spec.add_development_dependency 'pry-byebug', '3.1.0'
```

Comment:
@peterberkenbosch We're trying to limit the amount of deps, we already have `byebug` in `Gemfile`. :smile_cat: